packages/Search/test/tstDetailsTreeConstruction.cpp

```diff
@@ -47,7 +47,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( DetailsBVH, morton_codes, DeviceType )
                dtk::expandBits( k );
     };
     std::vector<unsigned int> ref( n,
-                                   Kokkos::ArithTraits<unsigned int>::max() );
+                                   std::numeric_limits<unsigned int>::max() );
```

Comment:
Operating on a `sed::vector` so did not bother adding a specialization and used `numeric_limits` instead